scripts/test_update_from_tag.sh

```diff
@@ -17,7 +17,7 @@ UPDATE_FROM_TAG=${UPDATE_FROM_TAG:-0.1.0}
 UPDATE_TO_IMAGE=${UPDATE_TO_IMAGE:-update_test}
 UPDATE_TO_TAG=${UPDATE_TO_TAG:-${GIT_ID}}
 DO_CLEANUP=${DO_CLEANUP:-true}
-PGOPTS="-v TEST_VERSION=${TEST_VERSION} -v TEST_REPAIR=${TEST_REPAIR} -v WITH_SUPERUSER=${WITH_SUPERUSER}"
+PGOPTS="-v TEST_VERSION=${TEST_VERSION} -v TEST_REPAIR=${TEST_REPAIR} -v WITH_SUPERUSER=${WITH_SUPERUSER} -v WITH_ROLES=true -v WITH_CHUNK=true"
```

Comment:
I don't really understand what `WITH_CHUNK=true` has to do with the description of this PR. Can you please explain what's the rationale, or is it maybe an unrelated fix?

Reply:
It is an omission from a previous commit. There are certain checks inside the update tests that work with individual chunks and there are not used with the smoke tests, but should be included when running a normal update test.

Reply:
It generates spurious errors when running the update tests making it harder to find the real problem.